.github/workflows/run-profiler-regression.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         runner-info: [
-          {arch: grayskull, runs-on: ["grayskull"]},
+          # No GS as tests now require synced starts. GS profiler tests will run on dedicated BMs
```

Comment:
Dedicated BMs? Mind clarifying?

Reply:
This is just explaining why there is no GS on the list of archs. 

Are you ok with 

 # No GS as tests now require synced starts. GS profiler tests need to run on dedicated BMs that do not use tt-smit reset

Reply:
I see - then should we make use of the no-smi-reset BM for this ? Are you asking us to add that declaration in here for you?

Reply:
I actually just added it to the top of the device perf regression nightly call. `Device perf regressions and output report`

Because we still have the worhole version on post commit. I am ok with checking GS only on nightlies. 

Do you believe putting it on that CI is confusing? The tests are still checking Device perf, just not in the context of fps and ops perf.

Reply:
Should I update the comment and mention that I have moved it to `Device perf regressions and output report`?

Reply:
No worries we can just resolve - did you just tag it with `models_device_performance_bare_metal` ?

I think that is a little bit confusing... I think it's ok for now I don't want to block you but can we open an issue to re-organize this and assign it to me + @ttmchiou ?

Reply:
https://github.com/orgs/tenstorrent-metal/projects/44/views/1?pane=issue&itemId=56978104 made this